sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/hashicorp.yaml

```diff
@@ -0,0 +1,32 @@
+provider:
+  metadata:
+    name: Hashicorp
+    category: Cloud provider
+    message: Make sure this Hashicorp token gets revoked, changed, and removed from the code.
+
+  rules:
+    - id: hashicorp-batch-token
+      rspecKey: S6773
+      metadata:
+        name: Hashicorp Batch tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - b.AAAAAQ
+              - hvb.AAAAAQ
+        matching:
+          pattern: "\\b((?:hv)?b\\.AAAAAQ[\\w-]{120,})"
```

Comment:
I think this is exactly the same as `\b((?:hv)?b\.AAAAAQ[\w-]{120})`, at least from a detection point of view. You should consider removing the `,` to avoid possible backtracking issues.

Reply:
Doing so will be less user-friendly (only a part of the token will be highlighted) so I keep it as it is.